package.json

```diff
@@ -35,5 +35,10 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "autoprefixer": "^10.4.8",
```

Comment:
please speak with Slav because he was thinking to add tailwind too - just tell him that you already did it